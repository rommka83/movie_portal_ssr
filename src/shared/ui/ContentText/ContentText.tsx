import React, { FC, HTMLAttributes } from 'react';
import styles from './contenttext.module.css';
import classNames from 'classnames';

export const ContentText: FC<HTMLAttributes<HTMLParagraphElement>> = ({
  className,
  children,
}) => {
  return <p className={classNames(styles.text, className)}>{children}</p>;
};
