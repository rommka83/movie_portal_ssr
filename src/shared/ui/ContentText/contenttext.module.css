.text {
  font-size: 15px;
  line-height: 22px;
  color: var(--gray-72);
}
