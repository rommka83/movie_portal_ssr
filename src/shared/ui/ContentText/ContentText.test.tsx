import renderer from 'react-test-renderer';
import { ContentText } from './ContentText';

describe('проверка ContentText', () => {
  test('отрисовка ContentText', () => {
    const component = renderer.create(
      <ContentText>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tenetur quod velit rem consequuntur nihil
        ullam, dicta modi nostrum dolore saepe illum provident quae asperiores excepturi possimus, fuga,
        distinctio officia numquam? Modi quam nihil sit atque vel facilis deserunt maiores maxime sed
        doloribus eos rerum corporis odio quae laudantium officia omnis nobis, hic in explicabo, ipsa
        veritatis ipsum distinctio ipsam! Mollitia? Quibusdam sed debitis, nisi minus dolores mollitia
        voluptatem laboriosam fugiat cumque, est quis? Tenetur ab dicta, repellendus eum, sunt saepe totam
        facere illo non aut explicabo et nulla! Aperiam, amet. Doloremque rem ipsa deleniti et sequi
        accusantium atque quis illo laborum dolore, alias dicta hic voluptas amet suscipit eaque, commodi, ut
        repellat dolorem. Exercitationem corporis ex ea dignissimos veniam quia.
      </ContentText>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
