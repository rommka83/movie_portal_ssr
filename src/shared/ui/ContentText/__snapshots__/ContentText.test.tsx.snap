// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`проверка ContentText отрисовка ContentText 1`] = `
<p
  className="text"
>
  Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tenetur quod velit rem consequuntur nihil ullam, dicta modi nostrum dolore saepe illum provident quae asperiores excepturi possimus, fuga, distinctio officia numquam? Modi quam nihil sit atque vel facilis deserunt maiores maxime sed doloribus eos rerum corporis odio quae laudantium officia omnis nobis, hic in explicabo, ipsa veritatis ipsum distinctio ipsam! Mollitia? Quibusdam sed debitis, nisi minus dolores mollitia voluptatem laboriosam fugiat cumque, est quis? Tenetur ab dicta, repellendus eum, sunt saepe totam facere illo non aut explicabo et nulla! Aperiam, amet. Doloremque rem ipsa deleniti et sequi accusantium atque quis illo laborum dolore, alias dicta hic voluptas amet suscipit eaque, commodi, ut repellat dolorem. Exercitationem corporis ex ea dignissimos veniam quia.
</p>
`;
